.buildkite/hooks/pre-command

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env sh
 set -eou pipefail
 
-if [ "$BUILDKITE_BRANCH" = "master" ]; then
-  branch_type="master"
+if [ "$BUILDKITE_BRANCH" = "main" ]; then
+  branch_type="main"
```

Comment:
The build role still includes `master` unless I'm mistaken

Reply:
If this is from `BIfS`, [my PR renamed it](https://github.com/cultureamp/base-infrastructure-for-services/pull/3082/files).

Reply:
But that being said, I will check if that's the case now.

Reply:
Ah, I see what you mean. By default the role is always going to be named `master`. Lemme fix that in a seperate commit for tracking reasons.

Reply:
Also, in the repo, it doesn't appear that BRANCH_TYPE is used anywhere else in the repo.